fibrechannel/fibrechannel.go

```diff
@@ -1 +1,300 @@
 package fibrechannel
+
+import (
+	"fmt"
+	"github.com/j-griffith/csi-connectors/logger"
+	"io/ioutil"
+	"k8s.io/kubernetes/pkg/util/mount"
+	volumeutil "k8s.io/kubernetes/pkg/volume/util"
```

Comment:
Same sort of note regarding imports, seems like we could just define the struct ourselves if we need it; but if we eliminate mount/fmt we don't need this one either.